gulp/typescript.js

```diff
@@ -53,4 +55,17 @@ module.exports = (gulp, plugins, blueprint) => {
             tsResult.dts,
         ]).pipe(blueprint.dest(project));
     });
+
+    const bundleTaskNames = blueprint.projectsWithBlock("typescript").map((project) => {
+        const taskName = `typescript-bundle-${project.id}`;
```

Comment:
and i'm disagreeing with both of those because it has everything to do with the typescript sources